### ISSUE TITLE
i have a questions for you

### ISSUE THREAD

**Issue by Juan-Cruz-Iturrioz** [Linaqruf/kohya-trainer#4]
Hello, I want to use the Kohya Trainer V4 
but I don't know how is the command to do a training with preservation of the original model

**Comment by Linaqruf** [comment]
You can't because you're doing training to insert new element to the model, that's what we called teacher learning, or in popular terms, fine-tune. But you can reduce the influence of new elements by adjusting the learning rate and reducing the max train step, or adding certain tagging to the dataset so that new elements will only appear when using certain prompts.

Or just do textual inversion on Auto1111's webui, you will get new file instead of modifying whole model.

Like this https://huggingface.co/Linaqruf/tarte, new elements will only appear when typing `scenery, no humans`

**Comment by Juan-Cruz-Iturrioz** [comment]
Thanks for the info and your time,
Person like you, that helping with a good explanation I like to study about models
